doc/admin/install/google_cloud.md

```diff
@@ -1,66 +1,101 @@
+   [GitHub releases](code-server-latest) page.
+3. Right click the Linux x64 `.tar.gz` release asset and copy the URL. In the
+   SSH terminal, run the following command:
+   ```
+   wget (paste the URL here)
+   ```
+4. Extract the downloaded file with the following command:
+   ```
+   tar -xvzf code-server*.tar.gz
+   ```
+5. Navigate to extracted directory with this command:
+   ```
+   cd code-server*/
+   ```
+6. Ensure the code-server binary is executable with the following command:
+   ```
+   chmod +x code-server
+   ```
+7. Finally, to start code-server run this command:
+   ```
+   ./code-server
+   ```
+8. code-server will start up, and the password will be printed in the output.
+   Make sure to copy the password for the next step.
+9. Open your browser and visit `https://$public_ip:8443/` (where `$public_ip`
+   is your Instance's public IP address). You will be greeted with a page
+   similar to the following screenshot. code-server is using a self-signed SSL
+   certificate for easy setup. In Chrome/Chromium, click **Advanced** then
+   click **proceed anyway**. In Firefox, click **Advanced**, then **Add
+   Exception**, then finally **Confirm Security Exception**.
+   <img src="../../assets/chrome_warning.png">
+
+[code-server-latest]: https://github.com/cdr/code-server/releases/latest
 
-- Start the code-server
-  ```
-  ./code-server
-  ```
-- Open your browser and visit `https://$public_ip:8443/` (where `$public_ip` is your Compute Engine instance's public IP address). You will be greeted with a page similar to the following screenshot. Code-server is using a self-signed SSL certificate for easy setup. In Chrome/Chromium, click **"Advanced"** then click **"proceed anyway"**. In Firefox, click **Advanced**, then **Add Exception**, then finally **Confirm Security Exception**.<img src ="../../assets/chrome_warning.png">
+---
 
-> For instructions on how to keep the server running after you end your SSH session please checkout [how to use systemd](https://www.linode.com/docs/quick-answers/linux/start-service-at-boot/) to start linux based services if they are killed
+### Post Installation Steps
 
----
+To ensure the connection between you and your server is encrypted, view our
+guides on [securing your setup](security-guide).
+
+For instructions on how to keep the server running after you end your SSH
+session please checkout [how to use systemd](systemd-guide). systemd will run
+code-server for you in the background as a service and restart it for you if it
+crashes.
 
-> NOTE: If you get stuck or need help, [file an issue](https://github.com/cdr/code-server/issues/new?&title=Improve+self-hosted+quickstart+guide), [tweet (@coderhq)](https://twitter.com/coderhq) or [email](mailto:support@coder.com?subject=Self-hosted%20quickstart%20guide).
+[security-guide]: ../../security/index.md
+[systemd-guide]: https://www.linode.com/docs/quick-answers/linux/start-service-at-boot/
```

Comment:
I suggest the [systemd guide from DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-configure-a-linux-service-to-start-automatically-after-a-crash-or-reboot-part-1-practical-examples) since its more comprehensive.